index.html

```diff
@@ -169,7 +169,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         <div class="row center-xs">
                             <div class="col">
                                 <h2>User Guides</h2>
-                                <p class="main-resources">Step-by-step guides to all things Monero are separated by category and cover everything from creating a wallet, recovering a wallet, and even how to edit this website.</p>
+                                <p class="main-resources">Step-by-step guides to all things Monero are separated by category and cover everything from creating a wallet to mnemonic seeds, and even how to edit this website.</p>
```

Comment:
This is the landing page for new users to Monero, who may also be newer to crypto, I think we should keep language simple. The moneropedia can use more direct terms.

Reply:
Good point, I agree.